DTS-STN/canadian-dental-care-plan
# AB#3022 - Handle document title in MetaFunction

### Description


Manage the document title within MetaFunction for enhanced flexibility. This approach allows for the document title to typically include the suffix `Canadian Dental Care Plan - Canada.ca` in English and `Régime canadien de soins dentaires - Canada.ca` in French, 99% of the time. MetaFunction possesses the capability to utilize loader data as required.

https://remix.run/docs/en/main/route/meta

### Related Azure Boards Work Items


[AB#3022](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3022)

### Screenshots (if applicable)


Splash Page
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/3c2b9ac9-3929-431e-92be-28c81db14cbf)

Apply - T&C Page
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/4fae5577-923d-4c14-a07f-6d5f5000c0e3)

Letters Page
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/1062c604-b501-4f8d-b8ff-24a5d34589f3)

### Checklist


### Test Instructions


Browser pages and the document title should be set and toggling the language should change it.

### Additional Notes


The document title content will be reviewed later.
